src/Components/Form/Form.js

```diff
@@ -0,0 +1,109 @@
+import React, { Component } from 'react';
+import { questionSet } from './questions'
+import './Form.scss';
+
+export class Form extends Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      allAnswers: [],
+      currentAnswers: [],
+      questionsPerActivity: [],
+      familyFriendly: true,
```

Comment:
We could remove `familyFriendly` from state, now that we are not planning to do the adult version.

Reply:
Another, agree 💯